src/data/pokemon/species_info.h

```diff
@@ -48948,6 +48949,7 @@ const struct SpeciesInfo gSpeciesInfo[] =
         .categoryName = _("Drummer"),                                           \
         .footprint = gMonFootprint_Rillaboom,                                   \
         LEARNSETS(Rillaboom),                                                   \
+        .formSpeciesIdTable = sRillaboomFormSpeciesIdTable,                     \
```

Comment:
I understand that they were missing from the base forms, but shouldn't they be in the Gigantamax forms also? The Mega Evolutions certainly seem to have the tables assigned to all forms, so we should probably be consistent (unless there's a reason not to be?)

Reply:
Yes, all forms of a species should have the table assigned.

Reply:
It's not at all clear here on GitHub, but this is a change to the `RILLABOOM_MISC_INFO` macro which is used by both `SPECIES_RILLABOOM` and `SPECIES_RILLABOOM_GIGANTAMAX`. So it should be applied to both the base form and the Gigantamax forms.

Reply:
Right, sorry, just woke up xD

Reply:
Apologies, my bad :) all good to go then!

Reply:
np! It's really difficult to see on GitHub!